tests/test-timber-comment.php

```diff
@@ -1,5 +1,9 @@
 <?php
 
+/**
```

Comment:
Are these temporary comments for working through the API or are they going to stay for 2.x, until we refactored the remaining classes, like `Comment`, `User`, etc.?

We could maybe make use of the `@todo` tag here. It’s getting picked up in a lot of IDEs, so it’s easier to work through.

Reply:
The `@group` tags are temporary, to allow e.g. `phpunit --group called-post-constructor`. I've replaced the `FIXME` tags with `@todo` - good call.